vendor.mod

```diff
@@ -145,7 +145,7 @@ require (
 	github.com/cenkalti/backoff/v4 v4.3.0 // indirect
 	github.com/cespare/xxhash/v2 v2.3.0 // indirect
 	github.com/cilium/ebpf v0.17.3 // indirect
-	github.com/container-storage-interface/spec v1.5.0 // indirect
+	github.com/container-storage-interface/spec v1.5.0
```

Comment:
just so I don't forget; looks like this import now has to be moved to the other section (for direct imports). Go doesn't always do a good job on that 😞

Reply:
Yes, seems to be hat vendor script does not handle it automatically. Fixed it now so we also get second round for CI to see that this does no introduce flaky issues.